modules/tts/tts-adapters/tts-adapter-cereproc/src/test/java/org/daisy/pipeline/tts/cereproc/impl/CereProcEngineTest.java

```diff
@@ -0,0 +1,77 @@
+package org.daisy.pipeline.tts.cereproc.impl;
+
+import net.sf.saxon.Configuration;
+import net.sf.saxon.s9api.XdmItem;
+import net.sf.saxon.s9api.XdmNode;
+import org.daisy.common.saxon.SaxonOutputValue;
+import org.daisy.pipeline.junit.AbstractTest;
+import org.daisy.pipeline.tts.Voice;
+import org.junit.Assert;
+import org.junit.Test;
+
+import javax.xml.stream.XMLStreamWriter;
+import java.io.File;
+import java.util.*;
+
+
+public class CereProcEngineTest extends AbstractTest {
```

Comment:
@carl-textalk Oh, one thing I wanted to say but forgot. I wonder whether the thing also works with OSGi. Would be good to know. Can you try it? Just tun the tests with the "test-with-paxexam" Maven profile. You probably need to override the `testDependencies` method in this class to something like:

```java
@Override
protected String[] testDependencies() {
	return new String[] {
		pipelineModule("tts-common"),
		pipelineModule("audio-common"),
		pipelineModule("audio-encoder-lame"),
		"com.ibu.uci:icu4j:?",
	};
}
```

Reply:
My two tests does not extend AbstractTest anymore.